tasks/release.py

```diff
@@ -235,25 +257,36 @@ def _get_highest_repo_version(auth, repo, new_rc_version, version_re):
     return highest_version
 
 
-def _get_highest_version_from_release_json(release_json, highest_major, version_re):
+def _get_highest_version_from_release_json(release_json, release_json_key, highest_major, version_re):
```

Comment:
nit: Maybe `release_json_key` should be an optional argument with default `None`?